scripts/configure_release_please/configure_release_please.py

```diff
@@ -75,7 +81,10 @@ def configure_release_please_manifest(
 
             gapic_version_file = next(package_dir.rglob("**/gapic_version.py"), None)
             if gapic_version_file is None:
-                raise Exception("Failed to find gapic_version.py")
+                if package_supports_gapic_version(package_dir):
+                    raise Exception("Failed to find gapic_version.py")
+                else:
+                    continue
```

Comment:
Hmm my thought was that we simply continue if `gapic_version.py` file doesn't exist instead of raising an exception + another check for a supported package. But if we do want to be explicit, how about we only check for a `gapic_version_file` if it is a supported package?

```suggestion
if not package_supports_gapic_version(package_dir):
    continue
gapic_version_file = next(package_dir.rglob("**/gapic_version.py"), None)
if gapic_version file is None:
    raise Exception("Failed to find gapic_version.py")
```

Reply:
Done in https://github.com/googleapis/google-cloud-python/pull/13436/commits/13bef06915147343e2c868b01169970c3fc7db79